docs/conops.md

```diff
@@ -0,0 +1 @@
+The Nightmare Scheduler program will let users input the times that they are available during the week and will come up with a time frame when everyone is available. There will be a simple GUI design, with three main buttons to start off with. The buttons will allow the user to enter in a new schedule, see which schedules have already been entered to avoid losing track, and generate a timeframe that works with everyone's schedule. When the button that adds a new schedule is clicked, the user will be prompted to enter in the person's name (this will be optional; if the user chooses not to add a name to the schedule, the third button that keeps track of the schedules that have already been entered will return blank) as well as the times that they are available. When all of the schedules have been added, the user can click the second button to generate a list of times during which everyone will be available.
```

Comment:
Recommend using some formatting.  For example.
```suggestion
The Nightmare Scheduler program will let users input the times that they are available during the week and will come up with a time frame when everyone is available. There will be a simple GUI design, with three main buttons to start off with. The buttons will allow the user to
 1) enter in a new schedule
 2) see which schedules have already been entered to avoid losing track
 3) generate a timeframe that works with everyone's schedule
## Add Schedule Button 
When the button that adds a new schedule is clicked, the user will be prompted to enter in the person's name (this will be optional; if the user chooses not to add a name to the schedule, the third button that keeps track of the schedules that have already been entered will return blank) as well as the times that they are available. When all of the schedules have been added, the user can click the second button to generate a list of times during which everyone will be available.
```